quiz.sql

```diff
@@ -7,29 +7,45 @@ CREATE DATABASE westeros;
 --   `people` has an id ( serial primary key), name (text), title (text), quote (text), image (text) and age (integer)
 
 -- your code starts here
-
+		create table houses (id serial primary key, name text, motto text, sigil text);
+		create table people (id serial primary key, name text, title text, quote text, image text, age integer);
 -- your code ends here
 
 -- Question 2. Insert the data from `westeros/houses.csv` into the `houses` table and the data from `westeros/people.csv` into the `people` table.
 
 -- your code starts here
-
+	LOAD DATA INFILE 'westeros/houses.csv' INTO TABLE houses
+  LOAD DATA INFILE 'westeros/people.csv' INTO TABLE people
 -- your code ends here
 
 -- Question 3. Write a query to get all attributes of people greater than a certain age *or* having a specific title.
 
 -- your code starts here
-
+SELECT name, motto, sigil
+FROM people
+WHERE age
+BETWEEN '10'
+AND '60'
+ORDER BY name;
 -- your code ends here
 
 -- Question 4. Change the motto of one of your houses. Look it up by its name.
 
 -- your code starts here
-
+UPDATE house SET motto = "weiner party"
+WHERE name = "House Lannister";
 -- your code ends here
 
 -- Question 5. Kill off, er, I mean delete one of the people. Look them up by their title.
 
 -- your code starts here
+DELETE FROM people
```

Comment:
There extra 'Where statements are redundant and will break your code. You can use 'OR' instead aka:

`DELETE FROM people WHERE name = 'Jon Snow' or name = 'Arya Stark';`